appcenter-link-scripts/package.json

```diff
@@ -18,7 +18,7 @@
   },
   "homepage": "https://github.com/microsoft/appcenter-sdk-react-native#readme",
   "dependencies": {
-    "debug": "2.2.0",
+    "debug": "2.6.9",
```

Comment:
In theory, the vulnerable package could be used in second level or third level dependencies in package.json. Therefore, to ensure we don't consume any of them, we should search for the unsecure version of the package in TestApp/package-lock.json and DemoApp/package-lock.json. 
Note that we don't checkin package-lock.json files but they are used by GitHub to detect vulnerabilities displayed in security alerts at https://github.com/microsoft/appcenter-sdk-react-native/network/alerts.

Reply:
I checked TestApp's package-lock.json and didn't find any reference to the insecure version of the package.

Reply:
So, did someone test react native link on a blank app RN 0.59?

Reply:
Tested with RN 0.59.9 blank app.